### ISSUE TITLE
findbugs fails

### ISSUE THREAD

**Issue by aegis123** [noveogroup/android-check#12]
I updated the plugin to 1.2.2 added findbugs config to the checks closure.

```
findbugs {
        skip true
        abortOnError false
        config easy()
        // directory for report files
        report new File(project.buildDir, 'reports/findbugs')
        // XML report file
        reportXML new File(project.buildDir, 'reports/findbugs/findbugs.xml')
        // HTML report file
        reportHTML new File(project.buildDir, 'reports/findbugs/findbugs.html')
}
```

and I get the following error

```
* What went wrong:
Execution failed for task ':app:androidFindbugs'.
> javax.xml.transform.TransformerException: javax.xml.transform.TransformerException:   com.sun.org.apache.xml.internal.utils.WrappedRuntimeException: Premature end of file.
```

Any idea how to fix this?

**Comment by igelgrun** [comment]
Could you rerun it using `./gradlew clean build --stacktrace`?

**Comment by aegis123** [comment]
that seems to run fine, it happens when i call ./gradlew clean check

**Comment by igelgrun** [comment]
fixed in 1.2.3
